pkgs/development/python-modules/oslo-utils/default.nix

```diff
@@ -67,12 +67,21 @@ buildPythonPackage rec {
     pyyaml
   ];
 
-  checkPhase = ''
+  disabledTests = [
```

Comment:
I did it before like https://github.com/NixOS/nixpkgs/blob/b36fb377314f39ad863408e5428a869666304b62/pkgs/development/python-modules/osc-lib/default.nix#L49-L54 but this is nicer and allows comments.

Reply:
I actually liked you're new way more than the old but whatever 😅